omniduct/filesystems/s3.py

```diff
@@ -0,0 +1,154 @@
+import os
+import re
+
+import pandas as pd
+
+from .base import FileSystemClient
+from ..utils.debug import logger
+
+
+class S3Client(FileSystemClient):
+
+    PROTOCOLS = ['s3']
+    DEFAULT_PORT = 80
+
+    def _init(self, bucket=None, aws_profile=None):
```

Comment:
can you add a comment / docs for what aws_profile is supposed to be?

Reply:
Sure. The aws profile is the configured name specified in one's ~/.aws/credentials file. Will add a comment to that extent. At some point I'll write full documentation for everything :).